src/cart/SConscript

```diff
@@ -97,6 +111,10 @@ def scons():
     prereqs.require(denv, 'mercury', 'pmix')
     denv.AppendUnique(RPATH=['$PREFIX/lib'])
 
+    compiler = env.get('COMPILER').lower()
+    if compiler == 'covc':
+        denv.Execute(SCons.Action.Action(cov_on, None))
```

Comment:
(pylint-no-member) Module 'SCons' has no 'Action' member

Reply:
do you need an else to turn it off?   Just thinking if the user happens to have it as their default compiler or something.  Otherwise, is off used anywhere?